other/build-icons.ts

```diff
@@ -0,0 +1,153 @@
+import { $ } from 'execa';
+import fsExtra from 'fs-extra';
+import { glob } from 'glob';
+import { parse } from 'node-html-parser';
+import * as path from 'node:path';
+
+const cwd = process.cwd();
+const inputDir = path.join(cwd, 'other', 'svg-icons');
+const inputDirRelative = path.relative(cwd, inputDir);
+const outputDir = path.join(cwd, 'app', 'components', 'ui', 'icons');
```

Comment:
```suggestion
const outputDir = path.join(cwd, 'app', 'components', 'icons');
```
i dont really see why we would need the "ui" subfolder since we are not using shadcn/ui like epic stack does. i would remove it

Reply:
Yep but it's juste a bit strange having the icon.tsx alone in the components folder

Reply:
aight, my bad, didnt think about this one. maybe a good idea to keep "generated" components in this ui subfolder and therefore we could move the "styled-components" inside here as well and leave the icons generation as it is?